source/minmax.js

```diff
@@ -0,0 +1,15 @@
+const minmax = numbers => {
+    numbers = numbers.split(' ').filter(num => num!=='');
```

Comment:
Давай приведем здесь форматирование в порядок:

```suggestion
    numbers = numbers.split(' ').filter(num => num !== '');
```